packages/maskbook/src/plugins/ITO/hooks/useRegion.ts

```diff
@@ -287,12 +299,10 @@ function createRegionResolver(api: string, field: string): RegionResolver {
     }
 }
 
-const IPGeoLocationAPI = createRegionResolver('https://api.ipgeolocationapi.com/geolocate', 'alpha2')
-// TODO the api https endpoint needs be paid
-const IPAPI = createRegionResolver('https://ip-api.com/json/?fields=status,countryCode', 'countryCode')
+const IPGeo = createRegionResolver('https://maskio.relaxgo.info:8443/geo', 'code')
```

Comment:
@yisiliu plz give him a aws account, run this program on aws lambda.

Reply:
the API is just for test, ignore it. I will rewrite the commit later

Reply:
@relaxgo plz create a new repo, publish the api code.

e.g: https://github.com/search?q=topic%3Alambda+org%3ADimensionDev+fork%3Atrue

Reply:
```suggestion
const IPGeo = createRegionResolver('https://service.r2d2.to/geolocation', 'region')
```